### PR TITLE
Docs: Improving the install with binary docs

### PR DESCRIPTION
**Pull request by 1Shubham7**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs update

* **What is the new behavior (if this is a feature change)?**

this is one among series of PRs planned to improve and rewrite some parts of documentation. Here I improved the "install with binary" docs. 

Works towards solving #533

### CODE REVIEW

**Comment by 1Shubham7** [comment]
@wbc6080 @fujitatomoya I am looking for your suggestions here as well.

**Review by fujitatomoya — COMMENTED** [review]
the fix lgtm with a couple of minor nitpics.

btw, why do we need to maintain? this is not even recommended procedure to install KubeEdge, what is the use case for this procedure? even with testing environment, we should not set up the KubeEdge with binaries? if that is so, probably we can just remove this page? i may be missing some cases, not sure about the history...

**Comment by 1Shubham7** [comment]
> the fix lgtm with a couple of minor nitpics.
> 
> btw, why do we need to maintain? this is not even recommended procedure to install KubeEdge, what is the use case for this procedure? even with testing environment, we should not set up the KubeEdge with binaries? if that is so, probably we can just remove this page? i may be missing some cases, not sure about the history...

@wbc6080  @fisherxu what are your views?

**Review by wbc6080 — COMMENTED** [review]
This part is mainly for developers. After modifying some codes of kubeedge, you can directly start the kubeedge cluster in binary mode for testing. I think it is necessary to keep this part, otherwise developers will not be able to check whether the subsequent kubeedge can work normally after the modifications.

File: docs/setup/install-with-binary.md [hunk 1]
```diff
@@ -2,16 +2,19 @@
...
 
 ## Setup Cloud Side (KubeEdge Master Node)
 
 ### Create CRDs
 
+Before setting up the cloud side, you need to create the necessary Custom Resource Definitions (CRDs) for KubeEdge. These CRDs define the custom resources that KubeEdge uses for managing devices, applications, and other components. To create the CRDs, run the following commands:
+
 ```shell
 kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1alpha2_device.yaml
 kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1alpha2_devicemodel.yaml
```
Comment (wbc6080) [inline docs/setup/install-with-binary.md:20]:
```suggestion
Before setting up the cloud side, you need to create the necessary Custom Resource Definitions (CRDs) for KubeEdge. These CRDs define the custom resources that KubeEdge uses for managing devices, applications, and other components. To create the CRDs, run the following commands:

```shell
kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1beta1_device.yaml
kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1beta1_devicemodel.yaml
```

Comment (wbc6080) [reply docs/setup/install-with-binary.md:20]: After release 1.15, we use the beta version of device crd, which can be updated here.

**Comment by 1Shubham7** [comment]
added all your suggestions, Please also confirm that are we only using beta version for those two crds or others as well.

**Review by wbc6080 — COMMENTED** [review]
/lgtm

**Comment by 1Shubham7** [comment]
@fujitatomoya @wbc6080 please approve this PR :)

**Review by fujitatomoya — APPROVED** [review]
lgtm. (i do not have the permission though...)

**Review by wbc6080 — APPROVED** [review]
You can wait for an approver to merge it

**Pull request closed without merging**